chart/cron-connector/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    # Original Helm labels v
```

Comment:
I would remove these labels are they are duplicates of the `app.kubernetes.io` ones.

Reply:
Can you tell which labels I should remove ? These are labels from kafka-connector.

Reply:
@stefanprodan PTAL

Reply:
We don't use the `app.kubernetes.io` labels in the OF chart, I would remove those for now.

Reply:
Done.